scripts/common/install_pippackages.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright (c) Contributors to the aswf-docker Project. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+BASEOS_MAJORVERSION=$(sed -n  's/^.* release \([0-9]*\)\..*$/\1/p' /etc/redhat-release)
+
+if [ "$BASEOS_MAJORVERSION" -gt "7" ]; then
+    python -m ensurepip --upgrade
+
+    # There is a package called python3-html5lib through yum/dnf but it install it
+    # for the system Python - which is not the same one that it used by downstream projects.
+    python -m pip install html5lib
+fi
```

Comment:
As discussed above, this adds a non-version source of dependencies. Also `html5lib` is a bit hidden in this install script. Would it make sense to have the script be more generic and install from a `requirements.txt` file?

Reply:
I agree that it is a bit hidden. We can also pin the version within the command itself (eg. `python -m pip install html5lib==1.1`), but it would still be hidden.

Were you thinking of adding a requirements.txt at the root of the repository? @jfpanisset